packages/core/src/odata-v2/index.ts

```diff
@@ -9,6 +9,6 @@ export * from './payload-value-converter';
 export * from './request-builder';
 export * from './selectable/custom-field';
 export * from './uri-conversion';
-export * from './legacy-request-configs';
+// export * from './legacy-request-configs';
```

Comment:
I don't remember the reason why these are legacy files.
If we decide to export the same classes from the common instead of the legacy folder, we might want to double check the compatibility, because we don't have confident test coverage for legacy code.
I would maybe at least include some updates in the compatibility notes.

Reply:
I guess you also want to delete this line?

Reply:
I exported them with a different name and added a compatibility  note.